src/DataDepsGenerators.jl

```diff
@@ -46,41 +80,51 @@ include("Figshare.jl")
 include("JSONLD/JSONLD.jl")
 
 function message(meta)
-    escape_multiline_string("""
-    Dataset: $(meta.fullname)
-    Website: $(meta.website)
-    $(meta.description)
-    """) |> strip
+    netString =  """
+    register(DataDep(
+        \"$(meta.shortname)\",
+        \"\"\""""
+    netString = format_meta(meta.website, netString, "Website")
```

Comment:
change these to:

```
netstring *= format_meta(meta.website, "Website")
```

and make `format_meta` return the string to be appended.
Or an empty string, if nothing is to be appended.

etc.
Then you don't need to keep passing in the